backend/src/main/java/wooteco/prolog/post/application/PostService.java

```diff
@@ -34,18 +36,35 @@ public PostService(PostDao postDao, MissionService missionService, TagService ta
                 .collect(Collectors.toList());
     }
 
+    public List<PostResponse> findWithFilter(List<Long> missions, List<Long> tags) {
```

Comment:
서비스에서는 `insertPosts`처럼 대상을(Post) 붙이는게 좋을 것 같아요 :)

```suggestion
    public List<PostResponse> findPostWithFilter(List<Long> missions, List<Long> tags) {
```